2dfft_utils/analysis/spiral_overlay.py

```diff
@@ -0,0 +1,276 @@
+'''
+    spiral_overlay.py
+    
+    DESCRIPTION:        Overlay spirals onto a FITS image.
+    
+    USAGE:              python spiral_overlay.py <filename>
+    
+    INPUTS:             Filename.
+                        FITS filename can be passed as a command line argument,
+                        but the program will prompt for it if it isn't.
+    
+    OUTPUT:             Interactive plot in python window.
+    
+    NOTES:              Uses pylab (including matplotlib and numpy) and astropy.
+                        All necessary libraries included in Ureka package.
+    
+                        Eqns used:
+    
+                            r = ae^(b*theta)
+    
+                            x = r*cos(theta)
+                            y = r*sin(theta)
+    
+                            tan(phi) = b
+                            phi = pitch angle
+    
+                        Edit theta_max, title, origin of the plot (imshow)
+                        & the colormap (imshow(...,cmap=..) as needed.
+                        Colormaps schemes available can be seen at:
+                        http://matplotlib.org/examples/color/colormaps_reference.html
+    
+                        Recommended cmap options are included as comments.
+                        
+                        It is recommended that you use cropped images with this program. There
+                        is a marked decrease in performance as image size increases.
```

Comment:
Specify that the program will run slower, unless you mean that other aspects may not work as well, either.